setup.py

```diff
@@ -135,7 +135,9 @@ def run(self):
             'acme_issuer = lemur.plugins.lemur_acme.plugin:ACMEIssuerPlugin',
             'acme_http_issuer = lemur.plugins.lemur_acme.plugin:ACMEHttpIssuerPlugin',
             'aws_destination = lemur.plugins.lemur_aws.plugin:AWSDestinationPlugin',
+            'aws_acm_destination = lemur.plugins.lemur_aws.plugin:ACMDestinationPlugin',
```

Comment:
Can we add some documentation and changelog entries for these plugins as well?

Reply:
Done!